src/css/base/typography.scss

```diff
@@ -4,7 +4,23 @@
 body {
   color: $color-base;
   font-weight: 400;
-  line-height: 1.7333333;
+  line-height: $leading-body;
+  font-size: $base-type-px;
+}
+
+a {
+  text-decoration: none;
```

Comment:
So would this mean that the only indication something is a link is a difference in color? I believe that might be an accessibility problem.